### PR TITLE
augustus update

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
`lpsolve` is missing and for boost please read our guidelines: https://bioconda.github.io/guidelines.html

**Comment by mictadlo** [comment]
@bgruening: I sent a [PR](https://github.com/conda-forge/staged-recipes/pull/2156) for `lp_solve`, but it failed because of `conda-forge-linter` and I am not sure how to fix it. By any chance, anyone knows how to fix this [PR](https://github.com/conda-forge/staged-recipes/pull/2156) ?

**Comment by bgruening** [comment]
@mictadlo the linter is giving you hints:
```
* The recipe could do with some maintainers listed in the extra/recipe-maintainers section.
* There should be one empty line at the end of the file.
```

**Comment by mictadlo** [comment]
@bgruening: 
* I checked and all files have one empty line at the end of each file. Therefore I do not understand the warning.
* Where can I found `extra/recipe-maintainers section?

**Comment by bgruening** [comment]
@mictadlo in all conda-forge recipes ;)
https://github.com/conda-forge/perl-feedstock/blob/master/recipe/meta.yaml#L32

**Comment by mictadlo** [comment]
@bgruening Thank you, your tip worked, but how can I now merge it now?

**Comment by bgruening** [comment]
@mictadlo you can't you need to wait on for the admins to merge it.

File: recipes/augustus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    - transMap2hints.pl --help 2>&1 | grep "convert transmap alignments to hints file"
+    - uniquePeptides.pl --help 2>&1 | grep "uniquePeptides.pl in.fa prefix > out.fa"
+    - utrgff2gbrowse.pl --help
+    - wigchoose.pl --help
+    - writeResultsPage.pl --help 2>&1 | grep "writeResultsPage ID species-name db-file grails-out www-out AugustusConfigPath AugustusScriptsPath final-flag"
+    - yaml2gff.1.4.pl --help 2>&1 | grep "< scipio.yaml > scipio.gff"
+
+about:
+  home: http://bioinf.uni-greifswald.de/augustus/
+  license: ''
```
Comment (bgruening) [inline recipes/augustus/meta.yaml]: http://bioinf.uni-greifswald.de/augustus/binaries/LICENCE.TXT
"GNU GENERAL PUBLIC LICENSE"

**Comment by bgruening** [comment]
@mictadlo general question, why do you not take the already existing recipe and try to enhance this to install the new version? Have things changed so dramatically?

**Comment by mictadlo** [comment]
The current one does not support OS X and it somehow does not require lp_solve. With my now I am getting on each platform different errors:

* MacOS [’gsl/gsl_matrix.h' file not found](https://travis-ci.org/bioconda/bioconda-recipes/jobs/190074800#L370)

* Linux: [/usr/bin/ld: cannot find -lcolamd](https://travis-ci.org/bioconda/bioconda-recipes/jobs/190074799#L617)

**Comment by jerowe** [comment]
@mictadlo , could you take a look here and see if this fits your requirements?

https://github.com/bioconda/bioconda-recipes/pull/3500

**Pull request closed without merging**